packages/p2p/src/stores/general-store.js

```diff
@@ -368,6 +369,25 @@ export default class GeneralStore extends BaseStore {
         this.order_table_type = order_table_type;
     }
 
+    @action.bound
+    setOrderTimeOut(time) {
+        const hours = Math.floor(time / 60);
+        const minutes = time % 60;
+        this.order_time_out = { hours, minutes };
+    }
+
+    @action.bound
+    setP2PConfig() {
+        requestWS({ website_status: 1 }).then(response => {
+            if (response) {
+                if (!response.error) {
+                    const { order_payment_period } = response.website_status?.p2p_config;
+                    this.setOrderTimeOut(order_payment_period);
+                }
+            }
```

Comment:
```suggestion
            if (response && !response.error) {
                const { order_payment_period } = response.website_status?.p2p_config;
                this.setOrderTimeOut(order_payment_period);
            }
```

Reply:
We used to take this approach. But later was decided to split the conditions.
@carol-binary?